Alignment/MillePedeAlignmentAlgorithm/test/alignment_cosmics_0T.py

```diff
@@ -0,0 +1,292 @@
+process.AlignmentTrackSelector.applyMultiplicityFilter = True# False
+process.AlignmentTrackSelector.maxMultiplicity = 1
+process.AlignmentTrackSelector.applyNHighestPt = False
+process.AlignmentTrackSelector.nHighestPt = 1
+process.AlignmentTrackSelector.seedOnlyFrom = 0
+process.AlignmentTrackSelector.applyIsolationCut = False
+process.AlignmentTrackSelector.minHitIsolation = 0.8
+process.AlignmentTrackSelector.applyChargeCheck = False # since no S/N cut is applied
+process.AlignmentTrackSelector.minHitChargeStrip = 50.
+
+# some further possibilities
+#process.AlignmentTrackSelector.applyChargeCheck = True
+#process.AlignmentTrackSelector.minHitChargeStrip = 50.
+# needs RECO files:
+#process.AlignmentTrackSelector.applyIsolationCut = True 
+#process.AlignmentTrackSelector.minHitIsolation = 0.8
+
+
+#-- new track hit filter
+# TrackerTrackHitFilter takes as input the tracks/trajectories coming out from TrackRefitter1
+process.load("RecoTracker.FinalTrackSelectors.TrackerTrackHitFilter_cff")
+process.TrackerTrackHitFilter.src = 'TrackRefitter1'
+process.TrackerTrackHitFilter.useTrajectories= True  # this is needed only if you require some selections; but it will work even if you don't ask for them
+process.TrackerTrackHitFilter.minimumHits = 8
+process.TrackerTrackHitFilter.commands = cms.vstring("keep PXB","keep PXE","keep TIB","keep TID","keep TOB","keep TEC")#,"drop TID stereo","drop TEC stereo")
+
+
+#placeholderdeadmodules
+
+
+
+process.TrackerTrackHitFilter.replaceWithInactiveHits = True
+process.TrackerTrackHitFilter.stripAllInvalidHits = False
+process.TrackerTrackHitFilter.rejectBadStoNHits = True
+process.TrackerTrackHitFilter.StoNcommands = cms.vstring("ALL 12.0")
+process.TrackerTrackHitFilter.rejectLowAngleHits = True
+process.TrackerTrackHitFilter.TrackAngleCut = 0.35# in rads, starting from the module surface; .35 for cosmcics ok, .17 for collision tracks
+process.TrackerTrackHitFilter.usePixelQualityFlag = True #False
+
+#now we give the TrackCandidate coming out of the TrackerTrackHitFilter to the track producer
+import RecoTracker.TrackProducer.CTFFinalFitWithMaterialP5_cff
+process.HitFilteredTracks = RecoTracker.TrackProducer.CTFFinalFitWithMaterialP5_cff.ctfWithMaterialTracksCosmics.clone(
+    src = 'TrackerTrackHitFilter',
+	NavigationSchool = cms.string(''),
+      TTRHBuilder = "WithAngleAndTemplate" #default
+)
+
+####################################################
+#-- Alignment producer
+####################################################
+process.load("Alignment.CommonAlignmentProducer.AlignmentProducer_cff")
+
+
+#placeholderalignables
+process.AlignmentProducer.checkDbAlignmentValidity = False
+
+#process.AlignmentProducer.doMuon = True # to align muon system
+
+process.AlignmentProducer.doMisalignmentScenario = True
+# If the above is true, you might want to choose the scenario:
+from Alignment.TrackerAlignment.Scenarios_cff import *
+process.AlignmentProducer.MisalignmentScenario = TECRing7Minus133mmScenario
```

Comment:
@eavdeeva @mschrode can you clarify if forcing the TEC R7 fix is still in use? I thought this was fixed already in cosmics